docs/preview/features/writing-different-telemetry-types.md

```diff
@@ -64,7 +64,7 @@ durationMeasurement.Start();
 var startTime = DateTimeOffset.UtcNow;
 
```

Comment:
I guess we'll have to update this once the other PR is in that adds the type of telemetry to the context